### PR TITLE
Add missing selectors for dropdown toggles

### PR DESCRIPTION
**Pull request by ibolton336**
Add dropdown toggle selectors for QE

### CODE REVIEW

File: pkg/client/src/app/pages/applications/analysis-wizard/set-mode.tsx [hunk 1]
```diff
@@ -66,8 +66,9 @@ export const SetMode: React.FunctionComponent<ISetMode> = ({
       </TextContent>
       <FormGroup label={t("wizard.label.analysisSource")} fieldId="sourceType">
         <SimpleSelect
+          toggleAriaLabel="analysis-mode-toggle"
           variant={SelectVariant.single}
-          aria-label="Select user perspective"
+          aria-label="analysis-mode-select"
```
Comment (mturley) [inline pkg/client/src/app/pages/applications/analysis-wizard/set-mode.tsx:71]: FYI, `aria-label` should be a human-readable sentence string, not an identifier. It's what gets read out by screen readers for blind users. "Select analysis mode" or something is actually best here.

For identifiers like this for QE, you can use the `toggleId` prop (SimpleSelect extends most props of PF's Select, and `toggleId` will put an id on the `<button>` used to toggle the select menu).

**Review by mturley — CHANGES_REQUESTED** [review]
Some feedback: we should leave `aria-label` as a human-readable phrase, and use `toggleId` for giving selectors to QE on SimpleSelect / Select.

File: pkg/client/src/app/pages/applications/analysis-wizard/set-mode.tsx (hunk 1 above)
Comment (ibolton336) [reply pkg/client/src/app/pages/applications/analysis-wizard/set-mode.tsx]: Thanks Mike - Updated to use toggleID for QE & fixed the aria-label.

**Review by mturley — APPROVED** [review]
LGTM, we can take a closer look at the rest of our aria-labels later on if we get to it.

**Pull request merged**